charm-slurmd/requirements.txt

```diff
@@ -1,3 +1,3 @@
 ops==1.3.0
 etcd3gw==1.0.2
-git+https://github.com/omnivector-solutions/slurm-ops-manager.git@0.8.4
+git+https://github.com/omnivector-solutions/slurm-ops-manager.git@0.8.4
```

Comment:
```suggestion
git+https://github.com/omnivector-solutions/slurm-ops-manager.git@0.8.5
```

Reply:
Could you update all other charms' requirements.txt as well?

Reply:
Yes, I gonna do that.